src/commands/synthetics/api.ts

```diff
@@ -121,7 +121,7 @@ const getBatch = (request: (args: AxiosRequestConfig) => AxiosPromise<{data: Ser
   const serverBatch = resp.data.data
 
   return {
-    results: serverBatch.results.filter((r) => r.status !== 'skipped') as Batch['results'],
+    results: serverBatch.results.filter((r) => r.status !== 'skipped' || r.selective_rerun) as Batch['results'],
```

Comment:
We still ignore skipped results in our code, but we now do an exception for `skipped` results _**with a `selective_rerun` property**_.

This is only possible when `decision: skip` and `reason: passed`:

https://github.com/DataDog/datadog-ci/blob/1d8e7821d132076a3237dc86f3bb185a355f3814/src/commands/synthetics/interfaces.ts#L163-L170